src/uct/ib/ud/base/ud_ep.c

```diff
@@ -1019,10 +1024,58 @@ void uct_ud_ep_process_rx(uct_ud_iface_t *iface, uct_ud_neth_t *neth, unsigned b
     ucs_mpool_put(skb);
 }
 
+static UCS_F_NOINLINE ucs_status_t
```

Comment:
IMO better remove UCS_F_NOINLINE and let compiler optimze

Reply:
done